frontend/setupChelonia.js

```diff
@@ -61,9 +61,6 @@ const setupChelonia = async (): Promise<*> => {
     }
   })
 
-  // this is to ensure compatibility between frontend and test/backend.test.js
-  sbp('okTurtles.data/set', 'API_URL', self.location.origin)
-
```

Comment:
Is this change intentional?

Reply:
Yes. This was moved to `sw-primary.js`

Reply:
That ensures both that it's called earlier and that it's not in `setupChelonia.js` (to avoid it being called twice if it was called by `main.js`.